lua/lazyvim/plugins/extras/coding/mini-snippets.lua

```diff
@@ -0,0 +1,159 @@
+--[[
+
+This extra for mini.snippets activates the following:
+1. During completion with either cmp or blink, expand lsp snippets
+2. To use other snippets: Press <c-j> in insert mode
+
+Note that there is no completion source. 
+Other snippets(custom or from friendly-snippets) are handled by pressing <c-j> in insert mode.
+
+In default LazyVim, neither cmp nor blink define the default mappings used by mini.snippets:
+- "<c-j>" to expand
+- "<c-l>" to jump next (dynamically created when in snippet context)
+- "<c-h>" to jump previous (dynamically created when in snippet context)
+
+It's difficult to have jump_next or jump_previous working in all cases when mapped to "<tab>"/"<s-tab>".
+For now, pressing tab inside a snippet just inserts a tab.
+LazyVim will warn the user when jump_next of jump_previous are overriden.
+
+Example override for your own config:
+return {
+  {
+    "echasnovski/mini.snippets",
+    opts = function(_, opts)
+      -- By default, for opts.snippets, the extra for mini.snippets only adds gen_loader.from_lang()
+      -- This provides a sensible quickstart, integrating with friendly-snippets
+      -- and your own language-specific snippets
+      --
+      -- In order to change opts.snippets, replace the entire table inside your own opts
+
+      local snippets, config_path = require("mini.snippets"), vim.fn.stdpath("config")
+
+      opts.snippets = { -- override opts.snippets provided by extra...
+        -- Load custom file with global snippets first (order matters)
+        snippets.gen_loader.from_file(config_path .. "/snippets/global.json"),
+
+        -- Load snippets based on current language by reading files from
+        -- "snippets/" subdirectories from 'runtimepath' directories.
+        snippets.gen_loader.from_lang(), -- this is the default in the extra...
+      }
+    end,
+  },
+}
+
+--]]
+
+local function expand(args)
+  ---@diagnostic disable-next-line: undefined-global
+  local insert = MiniSnippets.config.expand.insert or MiniSnippets.default_insert
+  insert({ body = args.body }) -- insert at cursor
+end
+
+local snippet_select_for_cmp = function(snippets, insert, opts)
```

Comment:
`select` step [takes only two `snippets` and `insert` arguments](https://github.com/echasnovski/mini.nvim/blob/733afb94ed88d32b839a644dc137604691e46032/doc/mini-snippets.txt#L626-L631). The third `opts` argument is only present in `default_select()` to be able to adjust its behavior inside custom `expand.select`.

Reply:
Thanks! Done in commit `6b5e0e3063c9`